### ISSUE TITLE
Origin Subscription Update

### ISSUE THREAD

**Issue by PioneerRaptor** [FriendsOfGalaxy/galaxy-integration-origin#18]
Wondering if the plugin is going to be updated to take advantage of GOG Galaxy 2.0 now allowing for subscriptions to be added.

If it has been updated for this, then I am not seeing any of my Origin Premier games appearing.

**Comment by FriendsOfGalaxy** [comment]
Yes, support for subscription was added. Please share your plugin logs, I'll take a look where is the problem https://github.com/FriendsOfGalaxy/galaxy-integration-origin/wiki/Log-files

**Comment by PioneerRaptor** [comment]
[plugin-origin-7f53219b-4e2b-4591-9f4f-dfc5f4ba9eb0.log](https://github.com/FriendsOfGalaxy/galaxy-integration-origin/files/4771714/plugin-origin-7f53219b-4e2b-4591-9f4f-dfc5f4ba9eb0.log)

My subscription tab in GOG is only showing Xbox Live Gold and Xbox Game Pass subscription games.

**Comment by FriendsOfGalaxy** [comment]
@PioneerRaptor  Right now i see that the plugin claims that no subscription is owned on the account

`2020-06-12 00:14:30,543 - root - DEBUG - Sending data: {"jsonrpc": "2.0", "id": "8658", "result": {"subscriptions": [{"subscription_name": "Origin Access Basic", "owned": false, "subscription_discovery": 3}, {"subscription_name": "Origin Access Premier", "owned": false, "subscription_discovery": 3}]}}`

can you confirm that you own an active access basic or premier subscription ? and if so provide details which one of them ? 

Maybe there is an unhandled edge case where for example if subscription is going to expire this month it might incorrectly think its already expired.

**Comment by PioneerRaptor** [comment]
@FriendsOfGalaxy I am an Origin Access Premier Subscriber and have been for almost 4 years now (obviously started out as just Access before Premier released). My current subscription ends on August 6, 2020. I re-up annually if that matters as well.

**Comment by FriendsOfGalaxy** [comment]
@PioneerRaptor thanks for info. I've prepared plugin version with extended logs. Could you please:
- Close Galaxy
- extract given zip directly to `%LocalAppData%\GOG.com\Galaxy\plugins\installed\` with overriding files
[origin more logs for subscription.zip](https://github.com/FriendsOfGalaxy/galaxy-integration-origin/files/4844862/origin.more.logs.for.subscription.zip)
- Open Galaxy, connect Origin if not connected yet, capture origin logs, paste here both `.log` and `.log.1`

**Comment by PioneerRaptor** [comment]
Thank you, I didn't know you could manually select subscriptions, so that's very helpful!

Log
[plugin-origin-7f53219b-4e2b-4591-9f4f-dfc5f4ba9eb0.log](https://github.com/FriendsOfGalaxy/galaxy-integration-origin/files/4847128/plugin-origin-7f53219b-4e2b-4591-9f4f-dfc5f4ba9eb0.log)

Log 1
[plugin-origin-7f53219b-4e2b-4591-9f4f-dfc5f4ba9eb0.txt](https://github.com/FriendsOfGalaxy/galaxy-integration-origin/files/4847138/plugin-origin-7f53219b-4e2b-4591-9f4f-dfc5f4ba9eb0.txt)

**Comment by FriendsOfGalaxy** [comment]
Thanks! Now I see where is the problem. We read only the first subscription entry found, that is already EXPIRED bue to transition to new subscription type in 2018.
https://gateway.ea.com/proxy/subscription/pids/2261665082/subscriptionsv2/1000002165082

I'll fix that

**Comment by PioneerRaptor** [comment]
You're awesome, thanks!
